plugins/ingress-nginx.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: ingress-nginx
+spec:
+  shortDescription: Interact with ingress-nginx
+  description: |
+    The official kubectl plugin for ingress-nginx.
+  version: 0.24.0
+  platforms:
+  - uri: https://github.com/kubernetes/ingress-nginx/releases/download/nginx-0.24.0/kubectl-ingress_nginx-darwin-amd64.tar.gz
+    sha256: 34e999ddc4e1926bbd7f4bdbb5fc40d6c3a5b2ffd711131e4f62e2d24cb2a179
+    files:
+    - from: "*"
+      to: "."
+    bin: "./kubectl-ingress_nginx"
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  - uri: https://github.com/kubernetes/ingress-nginx/releases/download/nginx-0.24.0/kubectl-ingress_nginx-linux-amd64.tar.gz
+    sha256: 05d48fe58faa0ff577fe3b9c90f2e21233a140712a9f9ff0c9bd867e65f1b4e5
+    files:
+    - from: "*"
+      to: "."
+    bin: "./kubectl-ingress_nginx"
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - uri: https://github.com/kubernetes/ingress-nginx/releases/download/nginx-0.24.0/kubectl-ingress_nginx-windows-amd64.tar.gz
+    sha256: 1850378a8f124633745f49f20ce8bd01ca0f0920646dc3be95332271ca0f8625
+    files:
+    - from: "*"
+      to: "."
+    bin: "./kubectl-ingress_nginx"
```

Comment:
I haven't tested this but it's a bit odd that the windows release is not distributed as .exe inside the tarball.

Krew will link it as .exe correctly, but I'm not sure if the symlink will work correctly because the target file isn't going to be named as .exe (see https://github.com/kubernetes-sigs/krew/issues/131). 

Basically, we don't know what happens on windows if we make a symlink to a source file named `foo` as `bar.exe`, and invoke `bar.exe`. Does `foo` actually get executed without needing an .exe extension?

Reply:
I just booted into windows to try it, and it seems like it won't even download it properly:

```console
 C:\Users\alex\.krew\bin\kubectl-krew.exe install --manifest ingress-nginx.yaml
Installing plugin: ingress-nginx
W0408 19:40:53.067709    4712 install.go:132] failed to install plugin "ingress-nginx": failed to dowload and move during installation: failed to move files: failed moving files: could not rename file from "C:\\Users\\alex\\AppData\\Local\\Temp\\krew-downloads\\ingress-nginx\\kubectl-ingress_nginx" to "C:\\Users\\alex\\AppData\\Local\\Temp\\krew-temp-move626560575\\kubectl-ingress_nginx": rename C:\Users\alex\AppData\Local\Temp\krew-downloads\ingress-nginx\kubectl-ingress_nginx C:\Users\alex\AppData\Local\Temp\krew-temp-move626560575\kubectl-ingress_nginx: The process cannot access the file because it is being used by another process.
F0408 19:40:53.067709    4712 root.go:52] failed to install some plugins: [ingress-nginx]
```

I installed `sniff` just fine, so I'm assuming it's the lack of a `.exe` extension. I might just leave Windows out at this point in time and fix this for the next release.

Reply:
That sounds like a krew bug. We are usually careful around Windows file opening semantics, but looks like there's a problem.
Do you mind filing a bug to https://github.com/kubernetes-sigs/krew/issues/new with `-v=10` flag? I wonder if other plugin installation are also failing.

Reply:
Just saw the second part of your command.
Can you try creating a .zip file with the windows binary (named *.exe) and run install again with `--archive foo.zip`? If this works, I'd be surprised, it doesn't explain the "file is used by another process" error.

Reply:
OK, let me know if you want me to go ahead and merge this.